app/views/users/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>All users</h1>
+        <h2>User name</h2>
+    <span>Number of posts:</span>
+        <h2>User name</h2>
+    <span>Number of posts:</span>
+        <h2>User name</h2>
+    <span>Number of posts:</span>
+        <h2>User name</h2>
+    <span>Number of posts:</span>
+</body>
+</html>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please remember that these views get yielded inside your layouts, in this case, your `application.html.erb`. This file already contains the `html` boilerplate, so defining it here is going to cause a bad markup for your page. Please consider only writing the portion related to this page, which is the title and the users! 🤓 🔧